cns/NetworkContainerContract_test.go

```diff
@@ -105,3 +105,40 @@ func TestNewPodInfoFromIPConfigsRequest(t *testing.T) {
 		})
 	}
 }
+
+func TestCreateNetworkContainerRequest_Validate(t *testing.T) {
```

Comment:
should we have another UT for postnetworkcontainerrequest validate?

Reply:
that really only tests that the for loop works as expected, but sure